src/slider/index.tsx

```diff
@@ -109,20 +111,31 @@ export const Slider = factory(function Slider({
 	const [{ output, label } = { output: undefined, label: undefined }] = children();
 
 	const { initialValue = min } = properties();
-	const existingInitialValue = icache.getOrSet('initialValue', min);
-	let value = icache.get('value') || initialValue;
-
-	if (initialValue !== existingInitialValue) {
-		value = initialValue;
-		if (initialValue > max) {
+	let { value } = properties();
+
+	if (!value) {
```

Comment:
The controlled value could be 0 though right?
```suggestion
	if (value === undefined) {
```